scrapy/downloadermiddlewares/httpcompression.py

```diff
@@ -8,12 +9,13 @@
 
 
 ACCEPTED_ENCODINGS = [b'gzip', b'deflate']
+logger = logging.getLogger(__name__)
 
 try:
     import brotli
     ACCEPTED_ENCODINGS.append(b'br')
 except ImportError:
-    pass
+    logger.error("brotli not installed")
```

Comment:
We shouldn't log an error here as this is not a problem

Reply:
Ok I thought this is an Import error, should I log a warning then?

Reply:
We still support configurations without brotli, please read the description of the needed changes in the original issue.